### DISCUSSION TITLE
GITHUB EDUCATION REFUSES MY APPLICATION REPEATEDLY!

### DISCUSSION THREAD

**Discussion by evencyan** [community/community#60736] [GitHub Education]
### Hi everyone,

I am a Chinese student. I meet the conditions of github education. I have sufficient conditions to prove that I am a student, but my application has been rejected repeatedly. I can provide proof materials. Please let me pass the application. Thank you

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open. You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
